packages/elements/src/components/ino-tooltip/ino-tooltip.tsx

```diff
@@ -62,6 +62,18 @@ export class Tooltip implements ComponentInterface {
    */
   @Prop() trigger: TooltipTrigger = 'mouseenter focus';
 
+  /**
+   * The delay in milliseconds before `ino-tooltip` shows up or hides.
+   * 
+   * If only one number is given, the show and hide delay get the given delay duration.
+   * 
+   * If two numbers are given e.g. `[500, 200]` the show delay is 500ms and the hide delay is 200ms.
+   * 
+   * A delay value of `null` defaults to 0ms.
+   * 
+   */
+  @Prop() delay?: number | [number, number | null];
```

Comment:
We could set `0` as the default value
```suggestion
  @Prop() delay?: number | [number, number | null] = 0;
```